src/Nova/Refund.php

```diff
@@ -68,12 +67,10 @@ public function fields(Request $request)
 
     protected function dataFields(): array
     {
-        return array_filter([
-            ID::make(),
-            nova('user') ? BelongsTo::make('Created By', 'creator', nova('user'))->exceptOnForms() : null,
-            DateTime::make('Created At')->exceptOnForms(),
-            nova('user') ? BelongsTo::make('Updated By', 'updater', nova('user'))->exceptOnForms() : null,
-            DateTime::make('Updated At')->exceptOnForms(),
-        ]);
+        return array_merge(
+            parent::dataFields(),
+            $this->creatorDataFields(),
+            $this->updaterDataFields(),
+        );
```

Comment:
@huntermontell Can you make sure all the other Nova resources have the dataFields section switched out to this new style?